pydpu/storage.py

```diff
@@ -137,8 +162,8 @@ def create(self, address):
                                 virtual_function=wrappers_pb2.Int32Value(value=2),
```

Comment:
should we use `self.pf` , `self.vf` and `self.port` here ?

Reply:
ooops... A mistake at interactive rebase. Fixed